aboutdialog.py

```diff
@@ -2,12 +2,20 @@
 
 # *****************************************************************************
 #
-# QConsolidate
+# OQ-Consolidate
 # ---------------------------------------------------------
-# Consolidates all layers from current QGIS project into one directory and
-# creates copy of current project using this consolidated layers.
+# OQ-Consolidate is a modified version of QConsolidate, that:
 #
-# Copyright (C) 2012-2013 Alexander Bruy (alexander.bruy@gmail.com)
+# * saves raster layers and converts vector layers to GeoPackage.
+# * creates a project if it does not exist yet
+# * allows to give a customized (validated) name to the consolidated project
+# * allows to store all the project files in a zip file
```

Comment:
Let's remove this

Reply:
We probably need also to change/add the `LICENSE` file in repo root to reflect AGPL 3 upgrade

Reply:
`LICENSE` done here https://github.com/gem/oq-consolidate/blob/programmatic-ui/LICENSE